bcda/client/bluebutton_test.go

```diff
@@ -452,8 +459,11 @@ func (s *BBRequestTestSuite) TestValidateRequest() {
 					"%s does not end with %s", req.Header.Get("BlueButton-OriginalUrl"), req.URL.String())
 				assert.Equal(t, req.URL.RawQuery, req.Header.Get("BlueButton-OriginalQuery"))
 
-				assert.Equal(t, jobID, req.Header.Get("BCDA-JOBID"))
-				assert.Equal(t, cmsID, req.Header.Get("BCDA-CMSID"))
+				assert.Equal(t, jobID, req.Header.Get(jobIDHeader))
+				assert.Equal(t, cmsID, req.Header.Get(clientIDHeader))
+				assert.Equal(t, "", req.Header.Get(oldJobIDHeader))
```

Comment:
nit (nitpick) - You can use `assert.Empty(t, req.Header.get(...))` instead

Reply:
ah, was following the pattern from a couple lines up, is Empty preferred?

Reply:
Yup, would you also mind changing the other ones to `assert.Empty`